FlashDevelop/Docking/DockablePanel.cs

```diff
@@ -22,7 +22,7 @@ public DockablePanel(Control ctrl, String pluginGuid)
             this.HideOnClose = true;
             this.Controls.Add(ctrl);
             Globals.MainForm.ThemeControls(this);
-            this.Show();
+            //this.Show();
```

Comment:
Seem to work fine with NOT showing panels by default and letting the layout handle it.